app/queries/content_items_query.rb

```diff
@@ -4,8 +4,12 @@ def self.build(options)
   end
 
   def results(options = {})
-    relation = if options[:organisation]
+    relation = if options[:organisation].present? && options[:taxonomy].present?
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/95/commits/b0d56da66f5118e261e0015f9a934a8baa42689a